doc/Language/regexes.pod6

```diff
@@ -39,25 +41,25 @@ Example of difference between C<m/ /> and C</ /> operators:
 Whitespace in literal regexes is generally ignored (except with the C<:s> or,
 completely, C<:sigspace> adverb).
 
-Comments work within a regular expression:
+Comments are allowed within a regular expression:
 
     / word #`(match lexical "word") /
 
 as long as the syntax for
 L<embedded comments|/language/syntax#Multi-line_/_embedded_comments>, with a
-backtick following the hash sign and enclosing delimiters, is used.
+backtick and enclosing delimiters following the hash sign, is used.
 
 =head1 Literals
 
-The simplest case for a regex is a match against a string literal:
+The simplest use case for a regex is a match against a string literal:
 
     if 'properly' ~~ / perl / {
         say "'properly' contains 'perl'";
     }
 
-Alphanumeric characters and the underscore C<_> are matched literally. All
-other characters must either be escaped with a backslash (for example, C<\:>
-to match a colon), or be within quotes:
+Alphanumeric characters, including the underscore C<_> which is considered
+alphabetic, are matched literally. All other characters must either be escaped
```

Comment:
This feels wrong, because it might give the impression that `dd "_" ~~ /<:alpha>/` would give a `Match`.  I would remove the "which is considered alphabetic" part.

Reply:
I understand what you are saying. At the same time, `"_".match(/ <alpha> /)' does yield a Match, which is why the current text confuses me.  But I see now that <alpha> is actually defined as "Alphabetic characters including _", so there is consistency. I'll revert the proposed change.